Source/WebKit/UIProcess/Extensions/WebExtensionContext.messages.in

```diff
@@ -48,6 +48,9 @@ messages -> WebExtensionContext {
     // Commands
     CommandsGetAll() -> (Vector<WebKit::WebExtensionCommandParameters> commands);
 
+    // DeclarativeNetRequest
+    DeclarativeNetRequestGetEnabledRulesets() -> (Vector<String> rulesetIdentifiers);
```

Comment:
This stays as-is, but the impl should be `const Vector<String>&` to avoid a copy.